app/models/language.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Language
+  def self.table_name_prefix
+    'language_'
+  end
```

Comment:
I didn't realize you could do this for inheriting the table prefix.  This isn't a change suggestion, but I saw this and wanted to give you a heads up in case you didn't know about a minor limitation:

https://stackoverflow.com/questions/27483859/nested-modules-in-rails-breaking-table-name-prefix